apps/src/sites/studio/pages/devise/shared/_oauth_links.js

```diff
@@ -0,0 +1,14 @@
+import $ from 'jquery';
+
+$(document).ready(() => {
+  $('.oauth_sign_in').click(window.dashboard.clientState.reset);
+  // If the makerBridge is available, we are in the Maker App, which has its
+  // own Google login button. Disabling this google-oauth button to avoid confusion.
+  if (!!window.MakerBridge) {
+    $('.with-google_oauth2').prop({
+      disabled: true,
+      title:
+        'Please use the ‘Log in with Google’ button in the toolbar at the top of the Maker App. If you do not see a ‘Log in with Google’ button, please download the latest Maker App version.'
```

Comment:
uh oh, I spy smart quotes! how about "Please use 'Log in ...' button ..." ? I know this was working before... but smart quotes inside single quotes seems like a find/replace error waiting to happen

Reply:
Good catch! I'll have to use a lint disable line for it

Reply:
Or you could keep the outer single quotes and use `\'` for the inner quotes, if that would avoid the lint error

Reply:
It still runs into a lint error because it wants to change `\'` into the special `‘` character instead